jena-text/src/test/java/org/apache/jena/query/text/TestTextGraphIndexExtra2.java

```diff
@@ -0,0 +1,335 @@
+/**
```

Comment:
Minor - license headers aren't javadoc so they should be `/*` not `/**`

The codebase all over the place seem to have quite a few `/**` - not just this PR.